src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -139,7 +149,23 @@ public Task Get()
                         .Concat(clusterTopology.Promotables.Keys)
                         .Concat(clusterTopology.Watchers.Keys)
                         .ToList();
+
                     allNodes.RemoveAll(n => databaseRecord.Topology.AllNodes.Contains(n));
+
+                    if (databaseRecord.Encrypted)
+                    {
+                        var nodeCount = allNodes.Count;
+                        var removed = allNodes.RemoveAll(n =>
```

Comment:
Can move this logic to the Linq expression above.
And you shouldn't care how much you removed, only that you have enough.